README.md

```diff
@@ -1,128 +1,128 @@
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-I would like to thank Microverse for their helpful practices.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+# My Social Media
+
+
+
+
+
+## Getting Started
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [My Social Media] <a name="about-project"></a>
+**[My Social Media]** The app that I created will allow you to: Add post, the post can have likes, comments and the user can have many likes, comments, posts.
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a>Ruby</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Using Ruby on rails]**
+- **[Using migrate]**
+- **[Using Database]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- the Ruby on rails and postgresql should be installed in your machine.
+- Run `bundle install`.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- `git clone https://github.com/najibullahjafari/my_social_media`
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Najibullah Jafari**
+
+- GitHub: [Najibullah_jafari](https://github.com/najibullahjafari)
+- Twitter: [Najibullah_jafari](https://twitter.com/Najib_Jafari_)
+- LinkedIn: [Najibullah_jafari](https://www.linkedin.com/in/najibullahjafari/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Future Features
+
+- I think about how to make it fast.
+- I will make nice ui for this.
+- I will do my best to follow DRY principle.
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/najibullahjafari/my_social_media/issues/).

```

Comment:
- [ ] It will be great to ensure that the link to the issues page is not broken. At the moment, a 404 error is seen. Kindly improve this.